plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java

```diff
@@ -53,7 +53,8 @@
 			ResourceKind.BUILD, 
 			ResourceKind.IMAGE_STREAM, 
 			ResourceKind.ROUTE,
-			ResourceKind.TEMPLATE
+			ResourceKind.TEMPLATE,
```

Comment:
just refactored this thing and noticed that there is a little difference in the arrays: there is `ResourceKind.IMAGE_STREAM_TAG` in ConnectionWrapper, but there is no such element in WatchManager's array. 
After i made them use the same array (including `ResourceKind.IMAGE_STREAM_TAG`) i started to get the following message in the console when expanding the service: 

> WARNING: A connection to https://10.1.2.2:8443/ was leaked. Did you forget to close a response body?

When i deleted `ResourceKind.IMAGE_STREAM_TAG` it became ok.
@jeffmaury @jcantrill so do we need this ResourceKind for both classes? Was it just missed in WatchManager before?